homeassistant/components/squeezebox_player/strings.json

```diff
@@ -0,0 +1,12 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "description": "Do you want to add Squeezebox Player to Home Assistant?"
```

Comment:
Not necessary as this is the reason people click that button.

Reply:
I shamelessly copied the config flow from the "radio browser" integration which asks that same question and has no further setup